modules/dxp/apps/analytics/analytics-reports-journal-impl/src/main/java/com/liferay/analytics/reports/journal/internal/info/item/JournalArticleAnalyticsReportsInfoItem.java

```diff
@@ -104,6 +104,19 @@ private Date _getJournalArticleFirstPublishLocalDate(
 		}
 	}
 
+	private Optional<User> _getUser(JournalArticle journalArticle) {
+		JournalArticle latestArticle =
+			_journalArticleLocalService.fetchLatestArticle(
+				journalArticle.getResourcePrimKey());
+
+		if (latestArticle == null) {
+			return Optional.empty();
+		}
+
+		return Optional.ofNullable(
```

Comment:
maybe you can use  Optional.ofNullable(_journalArticleLocalService.fetchLatestArticle(journalArticle.getResourcePrimKey()).
map(
latestJournalArticle -> _userLocalService.fetchUser(latestArticle.getUserId()
). (...)